specs/networking/libp2p-standardization.md

```diff
@@ -135,24 +161,369 @@ number of shard subnets is defined via `SHARD_SUBNET_COUNT` and the shard
 Each Gossipsub
 [Message](https://github.com/libp2p/go-libp2p-pubsub/blob/master/pb/rpc.proto#L17-L24)
 has a maximum size of 512KB (estimated from expected largest uncompressed block
-size).
+size). Clients SHOULD reject messages that are over this size limit.
```

Comment:
Yep I'll add this. I originally didn't because I was worried that with `and MUST NOT send` implementers may then assume no messages over the size limit will get sent and then not implement protective measures in the case that some do.